.vscode/launch.json

```diff
@@ -11,7 +11,7 @@
         "prompt",
         "print",
         "primary",
-        "--config=${workspaceRoot}/themes/default.omp.json",
+        "--config=~/.posh.omp.json",
```

Comment:
Just curious, why do we need this change?

Reply:
We don't. Will remove.